skore/src/skore/item/item.py

```diff
@@ -84,11 +84,22 @@ def __repr__(self) -> str:
         """Represent the item."""
         return f"{self.__class__.__name__}(...)"
 
-    @abstractmethod
-    def to_serializable(self):
-        """Va niquer ta mere."""
+    def _repr_mimebundle_(self, include=None, exclude=None):
+        del include, exclude
+        return {"text/html": self.as_html()}
+
+    def get_serializable_dict(self):
+        """Get a serializable dict from the item.
+
+        Derived class must call their super implementation
+        and merge the result with their output.
+        """
+        return {
+            "updated_at": self.updated_at,
+            "created_at": self.created_at,
+        }
 
```

Comment:
I'll try to summarize here our conversation. 

To me overriding a concrete method is something very classic in an OOP context. Calling the super implementation is somehow implicit has one wants to leverage the base implementation.

In our specific context `updated_at` and `created_at` instances variables are managed by the base class (`Item`), hence it is the responsibility of `Item` to implement how those variables goes in the dto output of `get_serializable_dict`. Here composition over inheritance mantra does not apply because we are not aggregating a different behavior on top of the base implementation, we enrich the outputted data with extra instance variables that may need to be in the resulting dto.

To me it's easier to apprehend for a new dev that if you want to create a new subclass of item you may (or not) override `get_serializable_dict` than searching for what basic information (nested in the base class) you must return in the dto.

On top of that third generation sub classes could benefit from there parent implementation as well (like SkrubTabelReportItem).